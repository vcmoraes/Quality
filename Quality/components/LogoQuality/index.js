import React from 'react'
import styled from 'styled-components'
import LogoQuality from '../../commons/images/quality-logo.png'

const Image = styled.Image`
  width: 100%;
  resize-mode: contain;
`
export default () => <Image source={LogoQuality} />
