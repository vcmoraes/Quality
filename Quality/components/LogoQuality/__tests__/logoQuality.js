import React from 'react'
import LogoQuality from '../index'
import renderer from 'react-test-renderer'

describe('Components - LogoQuality', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<LogoQuality />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
