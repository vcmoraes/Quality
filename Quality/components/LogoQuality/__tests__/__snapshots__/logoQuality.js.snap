// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - LogoQuality renders correctly 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../commons/images/quality-logo.png",
    }
  }
  style={
    Array [
      Object {
        "resizeMode": "contain",
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
